src/scss/_header.scss

```diff
@@ -131,9 +131,15 @@
 
   button[type='submit'] {
     background-color: $sttm-blue;
-
+    transition: 0.3s;
+    &:hover{
+      background-color: $sttm-orange;
+    }
     .dark-mode & {
       background-color: $sttm-blue;
+      &:hover{
+      background-color: $sttm-orange;
+    }
```

Comment:
We are writing the same css code for dark mode again, can we just skip it then. Ideally, it should work on both modes without rewriting.